### ISSUE TITLE
Inserting math commands should ensure presence of a math environment

### ISSUE THREAD

**Issue by ktraunmueller** [ktraunmueller/Compositor#110]
Double-clicking one of the Math toolbox items (e.g., `\alpha`) will insert the corresponding command into the source, regardless if the current scope is a math environment or not. On the next typesetting run, the math command will force TeX into math mode, which is probably not what you want.

→ Insert an inline math environment at the current caret location if there is none before inserting the math command

**Comment by ktraunmueller** [comment]
Haha, found that when investigating the issue:
```
private func ensureMathMode(at location: Int) {
    guard location != NSNotFound else {
        return
    }
    // TODO insert math shifts if not yet present
}
```

**Comment by ktraunmueller** [comment]
**Fixed** in 0.9.2
